server/scripts/geofenceToGeoJSON.js

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
Is this format commonly used in the pogo community? I thought we mostly just Poracle and GeoJson.

Reply:
No, mostly format will be Poracle and will remove this one. So we get one method.